src/@orb.yml

```diff
@@ -80,13 +87,10 @@ jobs:
           attach-workspace: true
           repo: <<parameters.ecr_repo>>
           tag: <<parameters.tags>>
+          dockerfile: <<parameters.dockerfile>>
```

Comment:
where is this parameter used?

Reply:
this parameter gets passed to the aws-ecr/build-and-push-image command. It is helpful when your Dockerfile is not named `Dockerfile` or not the default location